app/assets/stylesheets/custom.scss.erb

```diff
@@ -3652,10 +3652,10 @@ ul.progress-wrapper li.complete {
     list-style: none;
     /*border-left: 2px solid #007bc4;*/
     padding-left: 35px;
-    color: <%= EnrollRegistry[:qle_carousel].settings(:color).item %>;
+    color: var(--theme-primary-blue, <%= EnrollRegistry[:qle_carousel].settings(:color).item %>);
```

Comment:
Let's refactor the call to EnrollRegistry to use the color value directly as a fallback.  The CSS values stored in our Resource Registry .yml files are identical between clients (!!!) and we need to move away from this CSS in RR pattern altogether anyways.

Reply:
changed here: [f34b4b2](https://github.com/ideacrew/enroll/pull/3462/commits/f34b4b24c1539f37beae838d28a7522545d8ae4c)